src/controllers/TalksController.php

```diff
@@ -215,16 +211,11 @@ public function deleteAction($request, $db)
      */
     public function addTrackToTalk(Request $request, PDO $db)
     {
-        if (!isset($request->user_id)) {
-            throw new Exception("You must be logged in to create data", 400);
```

Comment:
Oddly a few of the existing requests had 400 response for this ? I had changed all of these to 401. Possible breaking change